.cargo/config.toml

```diff
@@ -1,6 +1,6 @@
 [build]
-target = "riscv32imc-esp-espidf"
-#target = "xtensa-esp32-espidf"
+# target = "riscv32imc-esp-espidf"
```

Comment:
Restore this to the original setup. You can always override with `cargo build ... --target ...`

Reply:
Restored